javafmllanguage/src/main/java/net/minecraftforge/fml/javafmlmod/FMLModContainer.java

```diff
@@ -48,7 +48,7 @@ public FMLModContainer(IModInfo info, String className, ModFileScanData modFileS
             var module = gameLayer.findModule(moduleName)
                 .orElseThrow(() -> new IllegalStateException("Failed to find " + moduleName + " in " + gameLayer));
             modClass = Class.forName(module, className);
-            LOGGER.trace(LOADING,"Loaded modclass {} with {}", modClass.getName(), modClass.getClassLoader());
+            LOGGER.debug(LOADING,"Loaded modclass {}/{} with {}", modClass.getModule().getName(), modClass.getName(), modClass.getClassLoader());
```

Comment:
I'd rather keep this as trace to avoid log spam. Log formatting suggestion to be clearer:
```suggestion
            LOGGER.trace(LOADING, "Loaded mod class {} from module {} with classloader {}", modClass.getName(), modClass.getModule().getName(), modClass.getClassLoader());
```

Reply:
Nothing prints out trace, so it's pretty much impossible to see. And I formatted it that way to mimic how vanilla java prints out the class names with modules. I'm gunna keep it as I think its kinda valuable information when debugging packs